outlier_detection/outlier_detect_ilf.py

```diff
@@ -0,0 +1,27 @@
+## Python verson 3.6
```

Comment:
Add code to check the version of Python being used.  Display an error and exit if the version doesn't meet minimum requirements.  (Hint: See `sys.version_info`)